mkdocs.yml

```diff
@@ -0,0 +1,5 @@
+site_name: TOOLNAME
+pages:
+  - Home: 'index.md'
```

Comment:
I think for most tools just 1 page is enough. For more complex tools we can add extra pages of course

Reply:
I agree, but this is a way to recycle the README already on the github page. So this prevents text duplication.
Maybe there is a more elegant way to do this with sphinx instead of mkdocs. (Sphinx can also process markdown)

Reply:
Well I think that a lot of the content in the readme can move to here anyway